### PR TITLE
Fixed confirmation bug, changed years, changed emails, added resume field, etc.

### PR DESCRIPTION
**Pull request by mecolmg**

### CODE REVIEW

File: app/server/services/stats.js [hunk 1]
```diff
@@ -104,7 +104,7 @@ function calculateStats(){
         newStats.confirmed += user.status.confirmed ? 1 : 0;
 
         // Count confirmed that are mit
-        newStats.confirmedMit += user.status.confirmed && email === "mit.edu" ? 1 : 0;
+        newStats.confirmedMit += user.status.confirmed && email === "vt.edu" ? 1 : 0;
```
Comment (sarj21) [inline app/server/services/stats.js]: should be confirmedVT probably. doesnt super matter anywhere it seems

**Review by sarj21 — CHANGES_REQUESTED** [review]
maybe change that one variable in stats.js

File: app/server/services/stats.js (hunk 1 above)
Comment (mecolmg) [reply app/server/services/stats.js]: Chnaged

**Review by sarj21 — APPROVED** [review]
thumbs up

**Pull request merged**